tests/test_charm.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2021 Canonical
+# See LICENSE file for licensing details.
+
+import unittest
+from unittest import mock
+from unittest.mock import call, mock_open, patch
+from tests.helpers import patch_network_get
+from charm import MongodbOperatorCharm
+from ops.model import ActiveStatus, MaintenanceStatus
+from ops.testing import Harness
+
+MONGO_CONF_ORIG = """# mongod.conf
+
+# for documentation of all options, see:
+#   http://docs.mongodb.org/manual/reference/configuration-options/
+
+# Where and how to store data.
+storage:
+  dbPath: /var/lib/mongodb
+  journal:
+    enabled: true
+#  engine:
+#  wiredTiger:
+
+# where to write logging data.
+systemLog:
+  destination: file
+  logAppend: true
+  path: /var/log/mongodb/mongod.log
+
+# network interfaces
+net:
+  port: 27017
+  bindIp: 127.0.0.1
+
+
+# how the process runs
+processManagement:
+  timeZoneInfo: /usr/share/zoneinfo
+
+#security:
+
+#operationProfiling:
+
+#replication:
+
+#sharding:
+
+## Enterprise-Only Options:
+
+#auditLog:
+
+#snmp:
+"""
+
+MONGO_CONF_ARGS = [
+    "net",
+    ":",
+    "\n",
+    "  ",
+    "bindIp",
+    ":",
+    " ",
+    "localhost,1.1.1.1",
+    "\n",
+    "  ",
+    "port",
+    ":",
+    " ",
+    "27017",
+    "\n",
+    "processManagement",
+    ":",
+    "\n",
+    "  ",
+    "timeZoneInfo",
+    ":",
+    " ",
+    "/usr/share/zoneinfo",
+    "\n",
+    "replication",
+    ":",
+    "\n",
+    "  ",
+    "replSetName",
+    ":",
+    " ",
+    "rs0",
+    "\n",
+    "storage",
+    ":",
+    "\n",
+    "  ",
+    "dbPath",
+    ":",
+    " ",
+    "/var/lib/mongodb",
+    "\n",
+    "  ",
+    "journal",
+    ":",
+    "\n",
+    "    ",
+    "enabled",
+    ":",
+    " ",
+    "true",
+    "\n",
+    "systemLog",
+    ":",
+    "\n",
+    "  ",
+    "destination",
+    ":",
+    " ",
+    "file",
+    "\n",
+    "  ",
+    "logAppend",
+    ":",
+    " ",
+    "true",
+    "\n",
+    "  ",
+    "path",
+    ":",
+    " ",
+    "/var/log/mongodb/mongod.log",
+    "\n",
+]
+
+
+class TestCharm(unittest.TestCase):
+    def setUp(self, *unused):
+        self.harness = Harness(MongodbOperatorCharm)
+        self.addCleanup(self.harness.cleanup)
+        self.harness.begin()
+        self.peer_rel_id = self.harness.add_relation("mongodb", "mongodb")
+        # self.harness.begin_with_initial_hooks()
+
+    @mock.patch("charm.MongodbOperatorCharm._add_mongodb_org_repository")
+    @mock.patch("charm.MongodbOperatorCharm._install_apt_packages")
+    def test_mongoDB_install(self, _add, _install):
```

Comment:
Here you can change to `test_mongodb_install` in order to follow the style guide.